test/e2e/apps/statefulset.go

```diff
@@ -510,6 +512,51 @@ var _ = SIGDescribe("StatefulSet", func() {
 
 		})
 
+		ginkgo.It("should perform canary updates and phased rolling updates of template modifications for partiton1 and delete pod-0 without failing container", func(ctx context.Context) {
+			ginkgo.By("Creating a new StatefulSet without failing container")
+			ss := e2estatefulset.NewStatefulSet("ss2", ns, headlessSvcName, 3, nil, nil, labels)
+			deletingPodForRollingUpdatePartitionTest(ctx, f, c, ns, ss)
+		})
+
+		ginkgo.It("should perform canary updates and phased rolling updates of template modifications for partiton1 and delete pod-0 with failing container", func(ctx context.Context) {
+			ginkgo.By("Creating a new StatefulSet with failing container")
+			ss1 := e2estatefulset.NewStatefulSet("ss3", ns, headlessSvcName, 3, nil, nil, labels)
+			ss1.Spec.Template.Spec.Containers = append(ss.Spec.Template.Spec.Containers, v1.Container{
```

Comment:
```suggestion
			ss.Spec.Template.Spec.Containers = append(ss.Spec.Template.Spec.Containers, v1.Container{
```
Or `ss1`, but it needs to be consistent throughout the test. I think `ss` it preferred to avoid confusion like here.

Reply:
sorry I missed that, now it's fixed.

Reply:
Thanks, feel free to mark resolved.